content/en/docs/components/pipelines/concepts/pipeline-root.md

```diff
@@ -45,13 +44,9 @@ If you want to specify the `pipeline root` to S3, please choose one of the follo
 ## How to configure pipeline root
 
 #### Via ConfigMaps
+The default Pipeline root at the Kubeflow pipeline deployment level by configuring the KFP Launcher configmap. 
```

Comment:
`The default Pipeline root at the Kubeflow pipeline deployment level by configuring the KFP Launcher configmap.`

What are you trying to say here exactly? Should this say 
`The default Pipeline root at the Kubeflow pipeline deployment level can be changed by configuring the KFP Launcher configmap. ` or something similar?

Reply:
yes, thanks, fixed